bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3524,7 +3525,8 @@ void ReplicaImp::onSeqNumIsStable(SeqNum newStableSeqNum, bool hasStateInformati
     IControlHandler::instance()->onStableCheckpoint();
     ControlStateManager::instance().wedge();
   }
-  onSeqNumIsStableCallbacks_.invokeAll(newStableSeqNum);
+  onSeqNumIsStableOneTimeCallbacks_.invokeAll(newStableSeqNum);
+  onSeqNumIsStableOneTimeCallbacks_.clear();
```

Comment:
adding clear changes the behavior of the callback mechanism ,but I guess that the name OneTime reflects that the change is desired